pkgs/applications/misc/jp2a/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, libjpeg }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.6";
+  name = "jp2a-${version}";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/project/jp2a/jp2a/${version}/${name}.tar.gz";
+    sha256 = "076frk3pa16s4r1b10zgy81vdlz0385zh3ykbnkaij25jn5aqc09";
+  };
+
+  makeFlags = "PREFIX=$(out)";
+
+  buildInputs = [ libjpeg ];
+
+  meta = with stdenv.lib; {
```

Comment:
Maybe when I have more free time.  I would not be able to take on that commitment.